wfe/wfe.go

```diff
@@ -1449,14 +1472,34 @@ func (wfe *WebFrontEndImpl) Authorization(ctx context.Context, logEvent *web.Req
 
 	response.Header().Add("Link", link(web.RelativeEndpoint(request, newCertPath), "next"))
 
-	err = wfe.writeJsonResponse(response, logEvent, http.StatusOK, authz)
+	err := wfe.writeJsonResponse(response, logEvent, http.StatusOK, authz)
 	if err != nil {
 		// InternalServerError because this is a failure to decode from our DB.
 		wfe.sendError(response, logEvent, probs.ServerInternal("Failed to JSON marshal authz"), err)
 		return
 	}
 }
 
+// Authorization is used by clients to submit an update to one of their
+// authorizations.
+func (wfe *WebFrontEndImpl) Authorization(ctx context.Context, logEvent *web.RequestEvent, response http.ResponseWriter, request *http.Request) {
+	// Requests to this handler should have a path that leads to a known authz
+	id := request.URL.Path
+	var authz core.Authorization
+	var err error
+	authz, err = wfe.SA.GetAuthorization(ctx, id)
```

Comment:
nit: why the explicit `var` for `authz` and `err` before the assignment instead of using:
```
authz, err := wfe.SA.GetAuthorization(ctx, id)
```

Reply:
Oops, missed this one. This code originally came from a big `if` switch on authzv1 vs authzv2, so the variables had to be declared outside the `if` scope.